Demo/Demo.scad

```diff
@@ -1,5 +1,5 @@
-// this changed from "use" to "include" with v0.3!!!
-include <Chamfers-for-OpenSCAD/Chamfer.scad>;
+// The use statement should be preferred over the include statement
+use <Chamfers-for-OpenSCAD/Chamfer.scad>;
```

Comment:
with "use" parts of the functionality don't work anymore. i cant remember which one, though.